plugins/git/octo.nix

```diff
@@ -0,0 +1,169 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "octo";
+  originalName = "octo.nvim";
+  defaultPackage = pkgs.vimPlugins.octo-nvim;
+
+  maintainers = [ helpers.maintainers.svl ];
+
+  extraPackages = [ pkgs.gh ];
+
+  settingsOptions = {
+    use_local_fs = helpers.defaultNullOpts.mkBool false ''
+      Use local files on right side of reviews.
+    '';
+
+    enable_builtin = helpers.defaultNullOpts.mkBool false ''
+      Shows a list of builtin actions when no action is provided.
+    '';
+
+    default_remote = helpers.defaultNullOpts.mkListOf types.str ''[ "upstream" "origin" ]'' ''
+      Order to try remotes
+    '';
+
+    ssh_aliases = helpers.defaultNullOpts.mkAttrsOf types.str "{ }" ''
+      SSH aliases.
+    '';
+
+    picker_config = helpers.mkCompositeOption "Picker configuration." {
+      use_emojis = helpers.defaultNullOpts.mkBool false ''
+        Use emojis in picker. Only used by "fzf-lua" picker for now.
+      '';
+
+      mappings =
+        let
+          mkMappingOption =
+            lhs: desc:
+            helpers.mkCompositeOption desc {
+              lhs = helpers.defaultNullOpts.mkStr lhs ''
+                Key to map.
+              '';
+
+              desc = helpers.defaultNullOpts.mkStr desc ''
+                Description of the mapping.
+              '';
+            };
+        in
+        helpers.mkCompositeOption "mappings for the pickers." {
+          open_in_browser = mkMappingOption "<C-b>" ''
+            Open issue in browser.
+          '';
+
+          copy_url = mkMappingOption "<C-y>" ''
+            Copy url to system clipboard.
+          '';
+
+          checkout_pr = mkMappingOption "<C-o>" ''
+            Checkout pull request.
+          '';
+
+          merge_pr = mkMappingOption "<C-r>" ''
+            Merge pull request.
+          '';
+        };
+    };
+
+    reaction_viewer_hint_icon = helpers.defaultNullOpts.mkStr "" ''
+      Marker for user reactions.
+    '';
+
+    user_icon = helpers.defaultNullOpts.mkStr " " ''
+      User Icon.
+    '';
+
+    timeline_marker = helpers.defaultNullOpts.mkStr "" ''
+      Timeline marker.
+    '';
+
+    timeline_indent = helpers.defaultNullOpts.mkStr "2" ''
+      Timeline indentation.
+    '';
+
+    right_bubble_delimiter = helpers.defaultNullOpts.mkStr "" ''
+      Bubble delimiter.
+    '';
+
+    left_bubble_delimiter = helpers.defaultNullOpts.mkStr "" ''
+      Bubble delimiter.
+    '';
+
+    github_hostname = helpers.defaultNullOpts.mkStr "" ''
+      Github Enterprise host.
+    '';
+
+    snippet_context_lines = helpers.defaultNullOpts.mkInt 4 ''
+      Number of lines around commented lines.
+    '';
+
+    gh_env = helpers.defaultNullOpts.mkAttributeSet "{ }" ''
```

Comment:
```suggestion
    gh_env = helpers.defaultNullOpts.mkAttributeSet { } ''
```

Reply:
i agree this is correct. just want to mention that chadtree, gitblame and diffview have it quoted